src/config/validate/runtime.ts

```diff
@@ -42,6 +42,30 @@ const validDistributionShares = Joi.custom((i) => {
 
 const validRpcUrl = Joi.string().uri({ scheme: ["https"] });
 
+const validPlugin = Joi.object({
```

Comment:
I am pretty sure we should not check plugin configuration strictly. It invalidates idea of plugin system.